website/docs/best-practices/custom-generic-tests.md

```diff
@@ -6,7 +6,7 @@ displayText: Writing custom generic tests
 hoverSnippet: Learn how to define your own custom generic tests.
```

Comment:
Thoughts on updating this page to "Writing custom generic data tests"?

Reply:
That works for me. I'll make the change